cookbook/upgrade/major_version.rst

```diff
@@ -45,17 +45,19 @@ in your browser, these notices are shown in the web dev toolbar:
 Deprecations in PHPUnit
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-By default, PHPUnit will handle deprecation notices as real errors. This means
-that all tests are aborted because it uses a BC layer.
+By default, deprecations that are triggered by Symfony are silenced. This means
+that, unless you use Xdebug or a custom error handler, your tests will still
+pass without a problem.
 
-To make sure this doesn't happen, you can install the PHPUnit bridge:
+If you want to make sure you don't use any deprecated features, you can install
+the Symfony PHPunit bridge:
 
 .. code-block:: bash
 
     $ composer require symfony/phpunit-bridge
```

Comment:
Sorry for commenting so late in a pull request labeled as `finished`, but I think this command should be:

```bash
$ composer require --dev symfony/phpunit-bridge
```